package/MDAnalysis/analysis/rdf_s.py

```diff
@@ -0,0 +1,194 @@
+
+    .. versionadded:: 0.13.0
+
+    """
+    def __init__(self, u, g1, g2,
+                 nbins=75, range=(0.0, 15.0), density=True, exclusion_block=None,
+                 **kwargs):
+        super(InterRDF_s, self).__init__(u.universe.trajectory, **kwargs)
+        self.g1 = []
+        self.g2 = []
+        for sel1 in g1:
+            self.g1.append(u.select_atoms(sel1))
+        for sel2 in g2:
+            self.g2.append(u.select_atoms(sel2))
+        self.u = u.universe
+        self._density = density
+
+        self.rdf_settings = {'bins': nbins,
+                             'range': range}
+        self._exclusion_block = exclusion_block
+
+    def _prepare(self):
+        # Empty list to store the RDF
+        count_list = []
+        count, edges = np.histogram([-1], **self.rdf_settings)
+        for n  in range(len(self.g1)):
+            count_list.append(np.zeros((len(self.g1[n]), len(self.g2[n]),
+                              len(count)), dtype=np.float64))
+        self.count = count_list
+        self.edges = edges
+        self.bins = 0.5 * (edges[:-1] + edges[1:])
+
+        # Need to know average volume
+        self.volume = 0.0
+
+        # Allocate a results array which we will reuse
+        self._result = np.zeros((len(self.g1), len(self.g2)), dtype=np.float64)
+
+        # exclusoin
+        # If provided exclusions, create a mask of _result which
+        # lets us take these out
+        #if self._exclusion_block is not None:
+        #    self._exclusion_mask = blocks_of(self._result,
+        #                                     *self._exclusion_block)
+        #    self._maxrange = self.rdf_settings['range'][1] + 1.0
+        #else:
+        #    self._exclusion_mask = None
+
+    def _single_frame(self):
+        for i in range(len(self.g1)):
+            self._result=distances.distance_array(self.g1[i].positions, self.g2[i].positions,
+                                                  box=self.u.dimensions)
+
+            # Maybe exclude same molecule distances
+            #if self._exclusion_mask is not None:
+            #   self._exclusion_mask[:] = self._maxrange
+
+
+            for j in range(len(self.g1[i])):
+                for k in range(len(self.g2[i])):
+                    count = np.histogram(self._result[j,k], **self.rdf_settings)[0]
+                    self.count[i][j,k,:] += count
+
+        self.volume += self._ts.volume
+
+
+    def _conclude(self):
+        # Volume in each radial shell
+        vol = np.power(self.edges[1:], 3) - np.power(self.edges[:-1], 3)
+        vol *= 4/3.0 * np.pi
+
+        # Empty lists to restore index, RDF and CDF
+        index = []
+        rdf_s = []
+        cdf_s = []
+
+        for i in range(len(self.g1)):
+
+            cdf_s.append(np.zeros(np.shape(self.count[i])))
+            for j in range(len(self.g1[i])):
+                for k in range(len(self.g2[i])):
+                    cdf_s[i][j,k,:] = np.array([sum(self.count[i][j,k,0:(n+1)])
```

Comment:
Instead of `sum` use `np.sum()` ... or I think you can simplify this expression by using [np.cumsum()](https://docs.scipy.org/doc/numpy/reference/generated/numpy.cumsum.html) (and get rid of the loop over `n`)